.github/workflows/terraform-integration-tests.yml

```diff
@@ -0,0 +1,101 @@
+name: Integration-tests
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    paths:
+      - '**.java'
```

Comment:
I think, using path like `./src` is better option. Also `pom.xml` changes should trigger the integration test setup as well, WDYT?

Reply:
I agree, I will add `pom.xml`. For `./src/` I thought only changes in the source files would be important enough to run the integration tests. But if you want I can change it to `./src`.

Reply:
`src` seems more convenient to me but `**.java` is ok for me as well, it is up to you 🙂

Reply:
I changed it to `src`.